_inc/client/components/upgrade-notice-content/index.jsx

```diff
@@ -38,62 +42,42 @@ const UpgradeNoticeContent = withModuleSettingsFormHelpers(
 		};
 
 		renderInnerContent() {
-			const blockEditorUrl = `${ this.props.adminUrl }post-new.php`;
+			/*eslint-disable react/no-danger*/
 			return (
 				<div className="jp-upgrade-notice__content">
-					<p>
-						{ __( 'The features you rely on, adapted for the new WordPress editor.' ) }
-						<br />
-						{ __( 'A new editor? Yes! {{a}}Learn more{{/a}}.', {
-							components: {
-								a: (
-									<ExternalLink
-										target="_blank"
-										rel="noopener noreferrer"
-										href={ 'https://wp.me/p1moTy-cee' }
-									/>
-								),
-							},
-						} ) }
-					</p>
-
-					<h2>{ __( 'Build your Jetpack site with blocks' ) }</h2>
-
-					<p>
-						{ __(
-							'Today, we are introducing the first wave of Jetpack-specific blocks built specifically ' +
-								'for the new editor experience: Simple Payment button, Form, Map, and Markdown.'
-						) }
-					</p>
-					<p>
-						<img
-							src={ imagePath + 'block-picker.png' }
-							width="250"
-							alt={ __( 'Jetpack is ready for the new WordPress editor' ) }
-						/>
-					</p>
+					<div dangerouslySetInnerHTML={ { __html: this.props.releasePostContent } } />
 					<div className="jp-dialogue__cta-container">
-						<Button primary={ true } href={ blockEditorUrl } onClick={ this.trackLearnMoreClick }>
-							{ __( 'Take me to the new editor' ) }
-						</Button>
 						<Button onClick={ this.dismissNotice }>{ __( 'Okay, got it!' ) }</Button>
 					</div>
+					<br />
+					<div>
+						<ExternalLink
+							href={ this.props.releasePostLink }
+							target="_blank"
+							rel="noopener noreferrer"
+						>
+							{ __( 'View this post on the Jetpack.com blog' ) }
+						</ExternalLink>
+						<br />
+						<ExternalLink
+							href="https://wordpress.org/plugins/jetpack/#developers"
+							target="_blank"
+							rel="noopener noreferrer"
+						>
+							{ __( 'View the Jetpack ' + this.props.version + ' changelog' ) }
```

Comment:
I think something like this would be easier to translate:

```suggestion
							{ __( 'View the Jetpack %(version)s changelog', {
								args: {
									version: this.props.version,
								},
								comment: '%(version) is a version number.',
							} ) }
```

Reply:
Thanks! I fixed this in commit 4d71be6.